server.go

```diff
@@ -2664,6 +2677,53 @@ func (s *Server) executeShowStatsStatement(stmt *influxql.ShowStatsStatement, us
 	return &Result{Series: rows}
 }
 
+func (s *Server) executeShowDiagnosticsStatement(stmt *influxql.ShowDiagnosticsStatement, user *User) *Result {
+	rows := make([]*influxql.Row, 0)
+
+	// Build information
```

Comment:
Much of the following is pretty manual. If there was a better way to do this (reflect?) I'd be up for it. It is clear as is though.